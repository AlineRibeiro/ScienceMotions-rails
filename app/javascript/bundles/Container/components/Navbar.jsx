import React from "react";

const NavBar = () => (
  <nav className="navbar navbar-inverse navbar-fixed-top" role="navigation">
    <div className="container">
      <div className="navbar-header">
        <button
          type="button"
          className="navbar-toggle"
          data-toggle="collapse"
          data-target="#bs-example-navbar-collapse-1"
        >
          <span className="sr-only">Toggle navigation</span>
          <span className="icon-bar" />
          <span className="icon-bar" />
          <span className="icon-bar" />
        </button>
        <a className="navbar-brand" href="#" />
      </div>
      <div
        className="collapse navbar-collapse"
        id="bs-example-navbar-collapse-1"
      >
        <ul className="nav navbar-nav">
          <li>
            <a href="#">Home</a>
          </li>
          <li>
            <a href="#">All lessons</a>
          </li>
          <li>
            <a href="#">Featured lessons</a>
          </li>
          <li>
            <a href="#">Team</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
);
export default NavBar;
