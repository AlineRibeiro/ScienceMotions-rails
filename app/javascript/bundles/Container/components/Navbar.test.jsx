import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

describe('Navbar', () => {
  window.scrollTo = jest.fn(() => ({})); 
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<Navbar />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})
