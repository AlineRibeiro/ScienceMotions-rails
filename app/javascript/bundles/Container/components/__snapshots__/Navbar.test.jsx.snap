// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly when there are no items 1`] = `
<nav
  className="navbar navbar-inverse navbar-fixed-top"
  role="navigation"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <button
        className="navbar-toggle"
        data-target="#bs-example-navbar-collapse-1"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
      <a
        className="navbar-brand"
        href="#"
      />
    </div>
    <div
      className="collapse navbar-collapse"
      id="bs-example-navbar-collapse-1"
    >
      <ul
        className="nav navbar-nav"
      >
        <li>
          <a
            href="#"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            All lessons
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Featured lessons
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Team
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
